Sources/Build/BuildOperationBuildSystemDelegateHandler.swift

```diff
@@ -644,10 +644,9 @@ final class CopyCommand: CustomLLBuildCommand {
 }
 
 /// Convenient llbuild build system delegate implementation
-final class BuildOperationBuildSystemDelegateHandler: LLBuildBuildSystemDelegate, SwiftCompilerOutputParserDelegate {
+final class LLBuildProgressTracker: LLBuildBuildSystemDelegate, SwiftCompilerOutputParserDelegate {
```

Comment:
@MaxDesiatov I think we should rename this whole file but there is so much stuff going on here that I don't know what a good name would be maybe something like LLBuildSupport...

Reply:
The file is big enough to split up into multiple files with names corresponding to names of files they contain

Reply:
I'll do this in a follow-up PR!